ui/packages/ui/src/app/Layout/appHeader.tsx

```diff
@@ -18,10 +23,33 @@ const AppHeader: React.FC<IAppHeader> = (props) => {
 	const homeClick = () => {
 		history.push("/")
 	}
-
+	const [isModalOpen, setIsModalOpen] = React.useState(false);
+	const handleModalToggle = () => setIsModalOpen(!isModalOpen);
+	const commitLink = `https://github.com/debezium/debezium-ui/commit/${process.env.COMMIT_HASH}`;
+	const Footer = () => (
```

Comment:
This Footer can be removed - no longer needed.

Reply:
opps, I totally thought I removed that!